### ISSUE TITLE
Vairy is inverted

### ISSUE THREAD

**Issue by DanteTorn** [heraldry/armory#7]
I noticed a difference between how Heraldicon and Drawshield displayed a blazon for Vairy so I inquired with a herald about it and they told me that the first tincture in a vairy blazon is the one touching the top.
Here, you can see that the second tincture is the one touching the top.
![image](https://github.com/heraldry/heraldicon/assets/35230199/6cd9cbf1-f4be-4ac2-890e-dac281904fb2)

**Comment by or** [comment]
Hmm, that sounds logical. I used Wikipedia for reference for that one, and that example used this approach. Or being the first complete bell in the top left. But it's quite true that in partitions generally the top left is the first.

I've been trying to find some definite rules for this, e.g. in https://en.wikisource.org/wiki/Page%3AA_Complete_Guide_to_Heraldry.djvu/102, but it doesn't seem to be explicitly covered. Another uncertain thing is how the pattern starts. Are the six bells the bells (like in your screenshot) or the upside down bells? So the pattern would start with the first tip right on the corner... but the tinctures would still have to be flipped in that case.

I found a bunch of examples where it is done like the above, but I think it's correct to swap the tinctures, so I'll look into that, cheers!
